packages/dropdowns/stories/examples/Menu/Advanced.tsx

```diff
@@ -46,29 +46,22 @@ interface IStoryProps {
   isAnimated: boolean;
   isCompact: boolean;
   isDisabled: boolean;
+  isOpen: boolean;
 }
 
 export const Advanced: Story<IStoryProps> = ({
   hasArrow,
   isAnimated,
   isCompact,
   placement,
-  isDisabled
+  isDisabled,
+  isOpen
 }) => {
-  const [isOpen, setIsOpen] = useState(false);
-
   return (
     <Grid>
       <Row style={{ minHeight: 400 }}>
         <Col textAlign="center">
-          <Dropdown
-            isOpen={isOpen}
-            onStateChange={changes => {
-              if (Object.prototype.hasOwnProperty.call(changes, 'isOpen')) {
-                setIsOpen(changes.isOpen === true);
-              }
-            }}
-          >
+          <Dropdown isOpen={isOpen || undefined}>
```

Comment:
I realize this treatment flips the component between controlled and uncontrolled (and results in a warning). But that the point – the control demonstrates both uses. More importantly, we have the ability to observe and use dev tools to debug the menu items which is imperative for the storybook demo.

Reply:
I think a more suitable structure would be to demonstrate one of each usage per story — similar to the [accordion](https://zendeskgarden.github.io/react-components/?path=/story/components-accordions-accordion--uncontrolled) stories. Probably not the right time for re-structuring, but just wanted to throw the idea out there.

Reply:
Perhaps. This directly copies the `Default` dropdowns story. So we may want to address both.